wiki/Article_Styling_Criteria/en.md

```diff
@@ -1,581 +1,559 @@
+- `18 June, 2013` must be `2013년06월18일`
+- `2007 March, 02` must be `2007年03月02日`
+
+### Language names
+
+When referring to the name of a language, capitalize the first letter of that language.
+
+- The `#spanish` chat channel are for those who speak **Spanish**.
+- The `#french` chat channel are for those who speak **French**.
+
+### Abbreviations
+
+When using abbreviations, it is really important to note what the abbreviation means upon first instance. For example:
+
+```
+The NC (Nightcore) mod is similar to the DT (Double Time) mod. While NC and DT increase the speed of the music by 50%, NC will change the pitch of the music and adds a clap and finish to each beat.
+```
+
+Abbreviations of osu! terms **must** be capitalised.
+For example:
+
+- `CS` for `Circle Size`
+- `AR` for `Approach Rate`
+- `DT` for `Double Time`
+
+When shortening the word "for example", use `e.g.` instead of `e.x.` or `i.e.`.
+
+### Serial comma
+
+_The serial comma is also known as the "Oxford comma" or the "Harvard comma"._
+
+When a list consists of 3 or more items in a sentence, use the serial comma. For example:
+
+- The game modes of _osu!_: osu!standard, osu!taiko, osu!catch`,` and osu!mania are fun to play with others.
+
+Keep in mind that there are rare expections to not using the serial commas.
+
+### Game modes
+
+When writting the name of the game modes, they are to be written as follows:
+
+- `osu!`
+  - `osu!standard` (unofficial, but used to prevent ambiguity)
+    - for folder names, it is required to use `osu!`, not `osu!standard`
+- `osu!taiko`
+- `osu!catch`
+- `osu!mania`
+
+You may use the old game mode names (e.g. `Catch the Beat`, `Taiko`, or `Mania`) only when discussing about said game mode's previous name.
+
+### Game modifiers
+
+Game modifiers **must** be capitalised. For example:
+
+- `Hard Rock`
+- `Nightcore`
+- `Double Time`
+- `Easy`
+
+### Gameplay elements
+
+Gameplay elements should **never** be capitalised. For example:
+
+- In osu!standard, **beatmaps** are composed of three different gameplay elements: **circles**, **sliders**, and **spinners**.
+- The **beatmap** **editor** is a place where **beatmappers** can map a song of their choice.
+
+### Contractions
+
+Contractions are a shortened form of a word or group of words.
+
+In the osu!wiki, contractions are **heavily** discouraged. This is to keep a formal tone in the osu!wiki.
+
+For example, use:
+
+- `Do not` instead of `Don't`
+- `You will` instead of `You'll`
+- `It is` instead of `It's`
+
+Note: the possessive forms of words (e.g. `the player's` or `the skinner's`) are not contractions.
+
+### Point of view
+
+When referring to the player, use `the player` or `a player`.
+
+**Never** use `he`/`his` or `she`/`her`! Instead, use `they`/`their`/`theirs`.
+
+In instances where you are referring directly to the reader (e.g. tutorial page), you could use the word `you`.
+
+
+**Never** use the first person perspective, `I`, or `I am`/`I'm`.

```

Comment:
Why did you remove `### Terminology`? It literally is just a section that shows how certain words are to be spelt.

Reply:
probably the rebase fucked it up, I'll recover that later